release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1419,14 +1419,38 @@ This caused the Ingress Operator to report the DNS status as `ready` when there
 [id="ocp-4-14-openshift-cli-bug-fixes"]
 ==== OpenShift CLI (oc)
 
-Previously, container image references that have both tag and digest were not correctly interpreted by the oc-mirror plug-in and resulted in the following error:
-
+* Previously, container image references that have both tag and digest were not correctly interpreted by the oc-mirror plug-in and resulted in the following error:
++
 [source,yaml]
 ----
 "localhost:6000/cp/cpd/postgresql:13.7@sha256" is not a valid image reference: invalid reference format
 ----
++
+This behavior has been fixed, and the references are now accepted and correctly mirrored.  (link:https://issues.redhat.com/browse/OCPBUGS-11840[*OCPBUGS-11840*])
+
+* Previously, you were receiving `401 - Unauthorized` error for registries where the number of path components exceeded the expected maximum path components. This issue is fixed by ensuring that the oc-mirror fails when the number of path components exceeds maximum path components. You can now set the maximum path components by using the flag `--max-nested-paths`, which accepts an integer value. By default, there is no limit to the maximum path components and is set to `0`. The generated `ImageContentSourcePolicy` will contain source and mirror references up to the repository level.
+(link:https://issues.redhat.com/browse/OCPBUGS-8111[*OCPBUGS-8111*]) (link:https://issues.redhat.com/browse/OCPBUGS-11910[*OCPBUGS-11910*]) (link:https://issues.redhat.com/browse/OCPBUGS-11922[*OCPBUGS-11922*])
+
+* Previously, oc-mirror flags `--short`, `-v`, and '--verbose' used to provide incorrect version information. You can now use the oc mirror `version' flag to know the correct version of oc-mirror. The oc-mirror flags `--short`, `-v`, and `--verbose` have been deprecated and will no longer be supported. (link:https://issues.redhat.com/browse/OCPBUGS-7845[*OCPBUGS-7845*])
+
+* Previously, mirroring from registry to disk would fail when several digests of an image are specified in the `imageSetConfig` without tags. The oc-mirror would add the default tag `latest` to the images. The issue is now fixed by using a truncated digest as the tag. (link:https://issues.redhat.com/browse/OCPBUGS-2633[*OCPBUGS-2633*])
+
+* Previously, oc-mirror would incorrectly add the Operator catalog to `ImageContentSourcePolicy`. This is an unexpected behavior because the Operator catalog is directly used from the destination registry through `CatalogSource`. This bug is fixed by ensuring that the oc-mirror does not add the Operator catalog as an entry to 'ImageContentSourcePolicy'. (link:https://issues.redhat.com/browse/OCPBUGS-10051[*OCPBUGS-10051*])
```

Comment:
Possible to find out what kind of thing `ImageContentSourcePolicy` is, or if it can be converted to plain text? Same with `CatalogSource`.

Edit looks like "`CatalogSource` resource" per a later bug.